tests/local_client_server_tcp_e2e/local_client_server_tcp_e2e.c

```diff
@@ -174,6 +192,7 @@ TEST_FUNCTION(client_and_server_connect_and_send_one_message_settled)
 
     // start the client
     SOCKETIO_CONFIG socketio_config = { "localhost", 5672, NULL };
```

Comment:
Then removing the initializers and doing a plain assignment is better, since otherwise there's 2 values set for the port.

---
In reply to: [109773444](https://github.com/Azure/azure-uamqp-c/pull/74#discussion_r109773444) [](ancestors = 109773444)